script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_6_1.txt

```diff
@@ -18,7 +18,7 @@ there is an all-black figure, all too strange to be real.
 <Page5>
 ...A man, probably in his thirties by the sound of his voice...walks into the living room steadily.
 The fact that he is all-black is not a metaphor.
-For whatever reason, he is covered in black bandages on not only his hands but all the way up to his face.
+For whatever reason, he is covered in black bandages not only his hands but all the way up to his face.
```

Comment:
You still need to have an `on` here -
> ... covered in black bandages not only **on** his hands but all the way ...

or you need to add a semicolon to break the sentence

> ... covered in black bandages; not only his hands, but all the way ....